EDDiscovery/EDDiscoveryForm.cs

```diff
@@ -235,10 +240,11 @@ private void EDDiscoveryForm_Shown(object sender, EventArgs e)
                 // Check for a new installer    
                 CheckForNewInstaller();
 
+                LogLine($"{Environment.NewLine}{Environment.NewLine}Loading completed!{Environment.NewLine}");
```

Comment:
I thought it might be nice to let the user now that we're done. Yeah, I know the indicator goes away, but sometimes I start hitting EDSM update and am left uncertain when one process stopped and the other started.

Ok, now I think about that update is probably threaded but probably still a nice extra indication.